AIMS/Controllers/Api/ReportsController.cs

```diff
@@ -452,6 +452,72 @@ public async Task<IActionResult> Download(int? id, CancellationToken ct = defaul
         return PhysicalFile(fullPath, "text/csv", fileDownloadName: _fs.Path.GetFileName(fullPath));
     }
 
+    [HttpGet("preview/{id:int}")]
```

Comment:
I've changed the column and storage in the latest PR; maybe something to change between the end of the sprint and the client meeting